spec/models/king_spec.rb

```diff
@@ -4,4 +4,20 @@
   it 'has type of King' do
     expect(subject.type).to eq 'King'
   end
+
+  describe '#valid_move?' do
+    it 'is a public instance method' do
+      expect(subject).to respond_to :valid_move?
```

Comment:
I tend to agree...I've only done that in the past because, to me, it helps improve the overall "narrative" and readability of the spec...as in, "ok, we are starting with a public method..."

Out it goes... :+1: :smile: